src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -248,21 +246,11 @@ protected void onStartup() {
      * Must be called after data source initialization.
      */
     private void checkToolRequirements() {
-        if ( requiresReads() && ! hasReads() ) {
```

Comment:
These are now enforced by the argument parsing system.  These checks are a hack to get around the fact that argument parsing didn't know how to deal with this.